frontend/src/components/bucket/BucketPermissionAddUser.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+
+import SearchUsers from '@/components/form/SearchUsers.vue';
+import { useBucketStore, useConfigStore } from '@/store';
+
+import type { User } from '@/interfaces';
+
+// Store
+const { permissions } = storeToRefs(useBucketStore());
+const { config } = storeToRefs(useConfigStore());
+
+// Functions
+const onAdd = (selectedUser: User | null) => {
+  if(selectedUser !== null) {
```

Comment:
Can't really tell from just a semantic level observation, but if we were to always force selectedUser to only be of type User, we wouldn't need to do that if `selectedUser !== null` check.

Reply:
I was keeping the function parameters as the same type as the state, but you're right, it's not required.